src/actions/menu_items_actions.ts

```diff
@@ -737,6 +737,7 @@ export const INSERT_LINK_NAME = (env: SpreadsheetChildEnv) => {
 //------------------------------------------------------------------------------
 
 export const FILTERS_CREATE_FILTER_TABLE = (env: SpreadsheetChildEnv) => {
+  env.model.selection.selectTableAroundSelection();
```

Comment:
I don't know we want to change the behaviour in 16.3 (probably not?), but in master the behaviour is that we only do `selectTableAroundSelection` if there is a single cell selected. Otherwise it's impossible to create a filter on half the columns of a range for example

Reply:
my bad, i'll try to go with the current master behaviour